app/vmctl/vm/vm.go

```diff
@@ -54,6 +55,13 @@ type Config struct {
 	RateLimit int64
 	// Whether to disable progress bar per VM worker
 	DisableProgressBar bool
+
+	// BackoffRetries defines how many retries we need to check if callback was successful.
+	BackoffRetries int
+	// BackoffFactor configure the length of delay after each failed attempt.
+	BackoffFactor float64
+	// BackoffMinDuration configure minimum (initial) repeat interval
+	BackoffMinDuration time.Duration
```

Comment:
Would it be easier to have a `backoff.Retry` object instead of these fields? In this way you won't need to describe these fields over and over for each new struct which needs retries.

Reply:
used only object